lib/BuildSystem/LaneBasedExecutionQueue.cpp

```diff
@@ -264,6 +264,7 @@ class LaneBasedExecutionQueue : public BuildExecutionQueue {
     int outputPipe[2]{ -1, -1 };
     if (shouldCaptureOutput) {
       if (::pipe(outputPipe) < 0) {
+        getDelegate().commandProcessStarted(context.job.getForCommand(), handle);
```

Comment:
Note this doesn't really have the intended effect -- the subprocess hasn't actually started yet still.